apps/experiments/views/experiment.py

```diff
@@ -977,7 +993,6 @@ def start_session_from_invite(request, team_slug: str, experiment_id: str, sessi
     if request.method == "POST":
         form = ConsentForm(consent, request.POST, initial=initial)
         if form.is_valid():
-            WebChannel.check_and_process_seed_message(experiment_session)
```

Comment:
Is this handled in the channel?

Reply:
Correct. When we start the session (this happens when we invite the user), the seed message is being generated, so doing it here essentially double generated the seed message